src/app/components/Home.module.css

```diff
@@ -14,14 +14,18 @@
 }
 
 .mainContentWrapper {
-  background-color: var(--otherWhite);
+  background-color: var(--grayBackground);
   display: flex;
   flex-direction: column;
   height: 100%;
   width: 100%;
 
+  &:global(.route__media),
```

Comment:
This was added temporarily, to keep the item page looking the same until we get around to those layout changes